domovoi.py

```diff
@@ -58,7 +58,7 @@ def run(self, processes_file):
         self.check_paths(solar_car_processes)
         self.start_processes(solar_car_processes)
         # Watch over the processes and respond accordingly
-        while len(solar_car_processes):
+        while len(solar_car_processes): # Domovoi continues to run as long as there are processes in the list, other wise it quits
```

Comment:
I think there should be a separate explicit check with an if statement. I think the intent will be much more clear that way, then we can avoid the comment.